### ISSUE TITLE
Android methods are not called

### ISSUE THREAD

**Issue by LEEY19** [OneSignal/react-native-onesignal#382]
This is a really weird issue as I have been using OneSignal on android without issues. At one point I realised my Android phones are not receiving notifications. After debugging, I found out that `OneSignal.configure({})` is probably not working. To verify, I tried this 
```
    OneSignal.getTags((receivedTags) => {
      console.log(receivedTags);
    });
```
on iOS, the console displays the tags, but nothing appears when I tried on Android. 

I tried removing the package and do `yarn install` again targeting the master branch directly, but issue still persists. Any hints on why this is happening would be very appreciated. @avishayil

**Comment by adouceur** [comment]
Same here. I've been using One Signal before with no issues but on my latest app with latest package version it's impossible for me to get anything to happen on Android and the device doesn't register.

**Comment by MelissaWeng** [comment]
Same here. Push notification works perfectly in development environment, but nothing happens in production environment. 
I use ExpoKit (ejected from Expo).

**Comment by nimeetshah0** [comment]
I'm having a similar problem. Package version 3.0.9, everything seems ok on iOS but on Android no listeners are fired and the device doesn't register.

**Comment by christianversloot** [comment]
Did anyone find a fix? Seems to happen here too. @geektimecoil

**Comment by christianversloot** [comment]
.... found a fix! In #396 

It's this (copy/pasted from there, all thanks to @yevhenjl):

In addition, if AndroidManifest.xml has the option tools:node="replace", OneSignal in Android does not work either. I just deleted this line and "react-native-device-info" and it all worked.

My manifest contained tools:node="replace", removing it worked!

**Comment by LEEY19** [comment]
Yes, confirmed that @christianversloot solution works, thanks!
